script/copy_config_bamboo.sh

```diff
@@ -13,3 +13,4 @@ cp /srv/apps/curate/curate_uc-STAGE/config/initializers/devise.rb.bamboo /srv/ap
 cp /srv/apps/curate/curate_uc-STAGE/config/environments/development.rb.bamboo /srv/apps/curate/curate_uc-STAGE/config/environments/development.rb
 cp /srv/apps/curate/curate_uc-STAGE/config/environments/production.rb.bamboo /srv/apps/curate/curate_uc-STAGE/config/environments/production.rb
 cp /srv/apps/curate/curate_uc-STAGE/config/authentication.yml.bamboo /srv/apps/curate/curate_uc-STAGE/config/authentication.yml
+cp /srv/apps/curate/curate_uc-STAGE/config/initializers/riiif.rb.bamboo cp /srv/apps/curate/curate_uc-STAGE/config/initializers/riiif.rb
```

Comment:
@scherztc Once more fix needed.  There is an extra `cp` in the middle of this line that needs to be removed.

Reply:
@hortongn -  This is ready to merge.